.github/CONTRIBUTING.md

```diff
@@ -371,13 +371,103 @@ This does not go for bug reports however, as we want to unearth and solve as man
 
 ### Commit
 
-* Focus on one thing per commit 
-* Keep changes as small as possible 
-* Only commit related functinoality for each commit 
-* Commit often
-* Strive for each commit to compile/build and pass tests 
-* Use [gitmoji](https://gitmoji.dev) when possible - point to VSCode extension and CLI tool
-* sidenote: when merging the CI pipeline combines each commit to a merge commit containing all commit messages
+You may have encountered statements like
+
+> Commit early, Commit often
+
+We agree. Apart from acting as checkpoints, good commits support collaboration by providing an overview of your changes and make navigating them easy. But only as long the commits are good.
+
+#### The good commit follows Git best practices
+
+You are encouraged to follow commonly adopted Git best practices. We would especially like to highlight:
+
+1. **Try to make each commit a logically separate changeset**
+    - Make changes in each commit _atomic_ &ndash; focus on one thing
+    - Keep your commits as small as possible, but keep your changes in each commit related
+    - If you have made multiple non-related changes and need to split them into separate commits you can use `git add --patch` to stage them
```

Comment:
Took a stab at this sentence - I think it is a bit hard to read: 
```suggestion
    - `git add --patch` can be used if you need to split non-related changes from a commit into separate commits.
```

I don't know if it's much better - I will let it be up to you 😄